gems/pending/util/miq-xml.rb

```diff
@@ -5,43 +5,34 @@
 require 'util/xml/xml_diff'
 require 'util/xml/xml_patch'
 
-class MiqXml  
-	#MIQ_XML_VERSION = 1.0
-	#MIQ_XML_VERSION = 1.1	# Added create_time to root in seconds for easier time conversions
-	MIQ_XML_VERSION = 2.0	# Changed sub-xmls, added namespaces
+class MiqXml
+  MIQ_XML_VERSION = 2.0 # Changed sub-xmls, added namespaces
 
-	@@defaultXmlType = :rexml  # REXML is always available so default it here.
+  @@defaultXmlType = :rexml  # REXML is always available so default it here.
 
-  # Now test to see if nokogiri is available
+  # Set to true if nokogiri is set as parser.
   @@nokogiri_loaded = false
-	begin

```

Comment:
XmlHash was originally written and used for fleecing, my guess is it could be replaced with Nokogiri but not sure the effort required.  It is also used in miq_request_workflow.rb in `get_ems_metadata_tree` and `convert_to_xml` where it is used to cache the layout of resources.  It would be harder to replace here since the MiqHashStruct objects set on element attributes stay intact and do not get converted to strings.

For the added use of `@@nokogiri_loaded` in the powershell code would it make more sense to do `xml.kind_of?(Nokogiri::XML::Document)` instead?